views/styles.html

```diff
@@ -329,6 +329,12 @@ <h1 class="base--h2">Form elements</h1>
   </section>
 
   <section class="_style-guide--group">
+    <h2 class="_style-guide--title">Alerts</h2>
+    {% from "components/alert.html" import alert %}
+    {{alert(alertData)}}
```

Comment:
if you're going to have the data be an array for this, you need your `for` statement here, not in the component